### ISSUE TITLE
Incorrect path for fonts being generated

### ISSUE THREAD

**Issue by akshaybabloo** [laynezh/vite-plugin-lib-assets#58]
I have gulp script that copies `woff` and `woff2` folder beside `src` folder. And in `src/main.scss` I imported a predefined css as URL which I got it from Fira Code font npm.

After adding the configuration for `libAssetsPlugin`:

```js
plugins: [
    Inspect(),
    libAssetsPlugin({
        name: '[name].[ext]',
        outputPath: 'fonts',
        include: /\.woff2?$/,
    }),
],
```

I get a generated output of css as:

```css
@font-face {
  font-family: 'Fira Code';
  src: url('.../fonts/FiraCode-Regular.woff2') format('woff2'),
    url("../fonts/FiraCode-Regular.woff") format("woff");
  font-weight: 400;
  font-style: normal;
}
```

The path for `woff` is correct, where the path for `woff2` is incorrect.

Please take a look at - https://github.com/akshaybabloo/gollahalli.com.

To build for above repository, please do

```sh
npm run static
npm run build:dev
```

The `static` copies required files to `themes/spark2` submodule and the `build:dev` script invokes a command under `themes/spark2` submodule. The file will be generated under - `themes/spark2/static/css/main.css`

I am not sure what wrong I have done.

**Comment by coder-layne** [comment]
@akshaybabloo Thank you for your feedback again. I have identified the cause of the problem and will fix it soon.

**Comment by coder-layne** [comment]
@akshaybabloo I just released v0.5.14, which resolves the issue on my machine.

**Comment by akshaybabloo** [comment]
@coder-layne That fixed it! Thank you very much!
